apps/aecontract/src/aect_contracts.erl

```diff
@@ -60,18 +61,20 @@
 
 -type amount() :: non_neg_integer().
 -type store() :: aect_contracts_store:store().
+-type serialization_version() :: 1 | 2.
 
 -record(contract, {
         %% Normal account fields
         id           :: aeser_id:id(),
         owner_id     :: aeser_id:id(),
         ct_version   :: version(),
-        code         :: binary(),     %% The byte code
+        code         :: {code, binary()} | {ref, aeser_id:id()}, %% the bytecode or DIRECT reference to contract with bytecode
         store        :: store(),      %% The current state/store (stored in a subtree in mpt)
         log          :: binary(),     %% The current event log
         active       :: boolean(),    %% false when disabled, but don't remove unless referrer_ids == []
         referrer_ids :: list(aeser_id:id()), %% List of contracts depending on this contract
-        deposit      :: amount()
+        deposit      :: amount(),
+        version      :: serialization_version()
```

Comment:
Why is version last not first @cytadela8?

Reply:
No reason. I appended the new field.